### PR TITLE
Distance along spline example

### PR DESCRIPTION
**Pull request by akaszynski**
Add an example based on this https://github.com/pyvista/pyvista/discussions/3123#discussioncomment-3321672 discussion.

### CODE REVIEW

File: examples/02-plot/distance-along-spline.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+
+###############################################################################
+# Determine the coordinates matching distance along a spline
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Here we write a simple function that gets the closest point matching a distance along a spline and then generate labels for those points.
+
+
+def get_point_along_spline(distance):
+    """Return the closest point on the spline a given a length along a spline."""
```
Comment (tkoyama010) [inline examples/02-plot/distance-along-spline.py:44]:
```suggestion
    """Return the closest point on the spline a given length along a spline."""
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

File: examples/02-plot/distance-along-spline.py (hunk 1 above)
Comment (adeak) [reply examples/02-plot/distance-along-spline.py:44]: Good catch, but I think this is supposed to be the other way around:
```suggestion
    """Return the closest point on the spline given a length along the spline."""
```

File: examples/00-load/create-spline.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
 """
-.. _ref_create_spline:
+.. _ref_create_spline_example:
```
Comment (akaszynski) [inline examples/00-load/create-spline.py:2]:
```suggestion
.. _create_spline_example:
```

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Pull request merged**